packages/app/src/components/nominationInfo/ApplicationForm.js

```diff
@@ -0,0 +1,311 @@
+import React, { useEffect, useRef, useContext, useState } from 'react';
+// import styles from './styles.module.css';
+import styles from './newstyles.module.css';
+import { useForm, Controller } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as Yup from 'yup';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+import { NominationsDataContext } from '../../utils/context/NominationsContext';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
+import DatePicker, { CalendarContainer } from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const ApplicationForm = props => {
+  const firstUpdate = useRef(true);
+
+  const [NominationsData, setNominationsData] = useContext(
+    NominationsDataContext
+  );
+
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+
+  const [admissionDate, setAdmissionDate] = useState(new Date());
+  const [dischargeDate, setDischargeDate] = useState(new Date());
+
+  useEffect(() => {
+    if (props.formData['Admission Date'] != 'Invalid Date') {
+      setAdmissionDate(props.formData['Admission Date']);
+    }
+  }, [props.formData['Admission Date']]);
+
+  useEffect(() => {
+    if (props.formData['Discharge Date'] != 'Invalid Date') {
```

Comment:
Minor nitpick, but I'd probably make a helper function for this if statement so the logic isn't repeated between both of the useEffects.

Reply:
this is file is being deleted. there are no user effects like this in the PR to be merged.